testing/test_runner.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+import context
+from roboplot.core.gpio import gpio_wrapper as gpio_wrapper
+
+
+class CustomTestRunner(unittest.TextTestRunner):
+    """ A test runner which behaves the same as the TextTestRunner, but which closes the emulator gui when all tests
+    have been run."""
+    def run(self, test):
+        result = super().run(test)
+        try:
+            gpio_wrapper.clean_up()
+        finally:
+            return result
```

Comment:
Not sure what this is trying to achieve, surely we want to try to run the tests and finally clean up the gpio pins?

(Sorry for duplication I only just figured out how to comment on a line!)